src/eos_kolibri/eos_kolibri_migrate/main.py

```diff
@@ -16,119 +16,250 @@
 import subprocess
 import sys
 
+from dataclasses import dataclass
 from pathlib import Path
 
-from . import config
-from .utils import argparse_dir_path, get_backup_path, recursive_chown
+from .. import config
+from ..utils import argparse_dir_path, get_backup_path, recursive_chown
 
-KOLIBRI_DATA_DIR = os.environ.get('KOLIBRI_HOME', config.KOLIBRI_DATA_DIR)
 
-# We ignore the following files from Kolibri's data directory:
-# - .data_version
-# - logs/
-# - server.pid
-# - sessions/
-# - static/
+ANSII_ESC = "\u001b"
+ANSII_BEL = "\u0007"
+ANSII_ESC_LINK = ANSII_ESC + "]8;;"
 
+
+# These files are sufficient to identify a Kolibri installation
 KOLIBRI_DATA_FILES = (
-    'content',
-    'db.sqlite3',
-    'job_storage.sqlite3',
-    'kolibri_settings.json',
-    'notifications.sqlite3'
+    "content",
+    "db.sqlite3",
 )
 
+
 def terminate_kolibri_server_process():
-    print("Stopping the Kolibri server...")
     kolibri_pids = []
     try:
-        kolibri_pids = subprocess.check_output(['/usr/bin/pgrep', '-x', 'kolibri']).split()
+        kolibri_pids = subprocess.check_output(
+            ["/usr/bin/pgrep", "-x", "kolibri"]
+        ).split()
```

Comment:
`killall kolibri` and :boom:

Reply:
Ah yes, much nicer. Alas, I realized this stuff is kind of pointless now since the process name probably isn't `kolibri` most of the time, but I'm not sure there's much we can do there. It _should_ be fine anyway, probably. It would be nice if we could do `flatpak kill org.learningequality.Kolibri` (or even find the process that owns the dbus name and kill _that_), but sudo wipes out the environment variables we would need for any of that so we'd need to kill Kolibri _then_ do privilege escalation, which is sounding complicated :/